src/vow.sol

```diff
@@ -50,22 +50,22 @@ contract Vow is LibNote {
     }
 
     // --- Data ---
-    VatLike public vat;
-    FlapLike public flapper;
-    FlopLike public flopper;
+    VatLike public vat;        // CDP Engine
+    FlapLike public flapper;   // Surplus Auction House
+    FlopLike public flopper;   // Debt Auction House
 
     mapping (uint256 => uint256) public sin; // debt queue
-    uint256 public Sin;   // queued debt          [rad]
-    uint256 public Ash;   // on-auction debt      [rad]
+    uint256 public Sin;   // Queued debt            [rad]
+    uint256 public Ash;   // On-auction debt        [rad]
 
-    uint256 public wait;  // flop delay
-    uint256 public dump;  // flop initial lot size  [wad]
-    uint256 public sump;  // flop fixed bid size    [rad]
+    uint256 public wait;  // Flop delay
+    uint256 public dump;  // Flop initial lot size  [wad]
+    uint256 public sump;  // Flop fixed bid size    [rad]
 
-    uint256 public bump;  // flap fixed lot size    [rad]
-    uint256 public hump;  // surplus buffer       [rad]
+    uint256 public bump;  // Flap fixed lot size    [rad]
+    uint256 public hump;  // Surplus buffer         [rad]
 
-    uint256 public live;
+    uint256 public live;  // Access Flag
```

Comment:
So it took me until the last file to realize this, but I really dislike "Access Flag" as the description for `live` variables. I think "Shutdown Flag" would be much more informative--do you agree?

Reply:
Yes, shutdown flag or caged flag is how I think about this.  Although, `live` is probably the most descriptive variable name in the entire codebase, we may not even need the comment.

Reply:
I like Shutdown Flag. When live = 0, we want it to undeniably be understood that shutdown has occurred.

Reply:
Yeah I would vote for changing to `// Shutdown Flag` globally. Or I guess since `1` means "not shut down", maybe it's really the `// Active Flag`...eh, I think I still like `// Shutdown Flag`. Or no comment is probably fine too.

Reply:
will go with `// Shutdown Flag`

Reply:
Well thinking that 1 indicates that the system is not caged, i'd rather go for the positive naming. So I prefer "Active Flag" than "Shutdown Flag" which would be the correct description for an `off` kind variable.
What about "Liveness Flag"? Would it make sense?